controllers/grid/settings/user/form/UserForm.php

```diff
@@ -62,12 +64,24 @@ public function initData()
 
             foreach ($userGroups as $userGroup) {
                 $userGroupIds[] = $userGroup->id;
-                $masthead[$userGroup->id] = Repo::userGroup()->userOnMasthead($this->userId, $userGroup->id);
             }
+
+            // Get user group IDs for user groups this user should not be displayed on the masthead for
+            // The others will be selected per default
+            $notOnMastheadUserGroupIds = UserUserGroup::query()
+                ->withUserId($this->userId)
+                ->withActive()
+                ->where('user_user_groups.masthead', UserUserGroupMastheadStatus::STATUS_OFF)
```

Comment:
It would be better to use a scoping function in `UserUserGroup` rather than exposing a `lower_camel_case` table name here and adding an explicit `where` clause. SQL building should be kept in the container.